### PR TITLE
Update version for  azure servicebus jms

### PR DESCRIPTION
**Pull request by chenrujun**

### CODE REVIEW

File: sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/utils/ApplicationId.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    //    kv: for Key Vault
+    //    sb: for Storage Blobs
+    //    sf: for Storage Files
+    //    eh: for Event Hub
+    //    bus: for Service Bus
+    //    cfg: for App Config
+    //    cos: for Cosmos
+    //    aad: for AAD
+    //    b2c: for AAD B2C
+    public static final String VERSION = Optional.of(ApplicationId.class)
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/utils/ApplicationId.java:24]: This is great.

File: sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java [hunk 2]
```diff
@@ -18,10 +18,10 @@
 
     private static final String QUEUE_NAME = "<ServiceBusQueueName>";
 
-    private final Logger LOGGER = LoggerFactory.getLogger(QueueReceiveController.class);
+    private final Logger logger = LoggerFactory.getLogger(QueueReceiveController.class);
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java:21]: Why don't we make this static?

File: sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/TopicReceiveController.java [hunk 3]
```diff
@@ -20,11 +20,11 @@
 
     private static final String SUBSCRIPTION_NAME = "<ServiceBusSubscriptionName>";
 
-    private final Logger LOGGER = LoggerFactory.getLogger(TopicReceiveController.class);
+    private final Logger logger = LoggerFactory.getLogger(TopicReceiveController.class);
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/TopicReceiveController.java:23]: Same here.

File: sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java (hunk 2 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java:21]: There is a lot of code like this:
```
private final Logger logger
```
If we modify, maybe we should modify together, in another PR.

File: sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/utils/ApplicationId.java (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/utils/ApplicationId.java:24]: Thank you.

File: sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java (hunk 2 above)
Comment (saragluna) [reply sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java:21]: Fair enough.

**Pull request merged**